File: codigos algoritmicos python/ejercicio21.py
print("clasificar edad")
edad= int(input("ingrese edad:  "))
if(edad in range(0,12)):
    print("niño")
elif(edad in range(13,17)):
    print("adolescente")
elif(edad in range(18,59)):
    print("adulto")
elif(edad in range(60,100)):
    print("adulto mayor")
else:
    print("Ya esta muerto")

File: codigos algoritmicos javascript/ejercicio21.js
console.log("clasificar edad");
let edad = 15;
if (edad >= 0 && edad <= 12) {
    console.log("Niño");
} else if (edad >= 13 && edad <= 17) {
    console.log("Adolescente");
} else if (edad >= 18 && edad <= 59) {
    console.log("Adulto");
} else if (edad >= 65 && edad <= 100) {
    console.log("Anciano");
} else {
    console.log("Ya estás muerto");
}